sockeye/coverage.py

```diff
@@ -33,15 +33,18 @@ class CoverageConfig(config.Config):
     Coverage configuration.
 
     :param type: Coverage name.
+    :param max_fertility: Maximum number of target words generated by a source word.
     :param num_hidden: Number of hidden units for coverage networks.
     :param layer_normalization: Apply layer normalization to coverage networks.
     """
     def __init__(self,
                  type: str,
+                 max_fertility: int,
```

Comment:
Could you make this a keyword argument with a default value (2) to ensure compatibility with older models using coverage?